apparmor.d/profiles-s-z/waybar

```diff
@@ -0,0 +1,45 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2024 odomingao
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/3.0>,
+
+include <tunables/global>
+
+@{exec_path} = @{bin}/waybar
+
+profile waybar /{,usr/}{,s}bin/waybar flags=(attach_disconnected) {
+  include <abstractions/audio>
+  include <abstractions/base>
+  include <abstractions/dconf-write>
+  include <abstractions/desktop>
+  include <abstractions/nameservice-strict>
+
+  network inet dgram,
+  network inet6 dgram,
+  network netlink raw,
+
+  @{exec_path} mr,
+
+  @{bin}/** rPUx,
+  @{user_bin_dirs}/** rPUx,
+
+  /usr/share/icons/** r,
+  /usr/share/pixmaps/** r,
+
+  owner @{HOME}/.icons/** r,
+  owner @{HOME}/.themes/** r,
```

Comment:
These are included in `abstractions/desktop`. Same for `owner @{user_share_dirs}/icons/** r,`